Design/Vehicle-Sizing/Engine Sizing/Instructions.txt

```diff
@@ -0,0 +1,13 @@
+Based on openrocketengine, a program designed by Cameron Flannery
```

Comment:
You shouldn't just copy the whole library. Use the library in your custom scripts. I.e.
`import openrocketengine as ore` at the top of the file.